assets/less/code.less

```diff
@@ -1,138 +1,127 @@
-/*
-
-github.com style (c) Vasily Polovnyov <vast@whiteants.net>
-  Released under the same license as highlight.js
-  https://github.com/isagalaev/highlight.js/blob/master/LICENSE
-
-Source: https://github.com/isagalaev/highlight.js/blob/master/src/styles/github.css
```

Comment:
Please, keep the header and update the reference to the original file. Is [this](https://raw.githubusercontent.com/isagalaev/highlight.js/master/src/styles/tomorrow.css) right?